CIP-0072/README.md

```diff
@@ -0,0 +1,613 @@
+---
+CIP: 72
+Title: Cardano dApp Registration & Discovery
+Status: Proposed
+Category: Metadata
+Authors: 
+  - Bruno Martins <bruno.martins@iohk.io>
+  - Mateusz Czeladka <mateusz.czeladka@cardanofoundation.org>
+Implementors: []
```

Comment:
Given that there's an implementation plan, this shouldn't be empty.

Reply:
done